docs/versions/21.0.md

```diff
@@ -0,0 +1,25 @@
+    While we have resolved many errors during this work, it is possible that new errors were introduced. If you find an issue, please [contact us](/#contact-us).
+
+## 21.0.0 (ETA 2024-01-08)
+
+### Major Changes
+
+- **Clarified our name** The preferred name is now "HGVS Nomenclature". The many previous names (e.g., HGVS guidelines, HGVS recommendations, VarNomen, Mutnomen) created confusion and are obsolete.
+- **Chose a stable URL** To foster stable URLs and better serve the community, HGVS Nomenclature now has a dedicated domain name, [hgvs-nomenclature.org](hgvs-nomenclature.org). The previous pages are now archived at [archive.hgvs-nomenclature.org](https://archive.hgvs-nomenclature.org/). The old site, [varnomen.hgvs.org](https://varnomen.hgvs.org/) forwards users to the corresponding page on the new site.
+- **Modernized our documentation system** Pages from the old documentation were migrated to a system that should make it easier to maintain HGVS Nomenclature and support multiple device size (phones, tablets, laptops). The intention during migration was to retain the intention of the original text, while modernizing the management of HGVS Nomenclature and cleaning up the presentation.
+- **Switched versioning schemes** HGVS Nomenclature will now use Semantic Versioning, which should make it easier for users to track new features in HGVS Nomenclature. Beginning with this release, historical versions will also be available adjacent to newer versions. See [Versions](..) for details.
+- **Joined HUGO** HGVS Nomenclature is overseen by the [Human Variant Nomenclature Committee](/hvnc/), a subcommittee of the [Human Genome Organization (HUGO)](https://www.hugo-international.org/).
+- **Modernized mechanism to contact the HVNC** We have introduced two new mechanisms to contact the HVNC: there is now an email list to discuss HGVS nomenclature and we have formalized the process by which may submit ideas. See [Contact Us](/#contact-us) for details. The Facebook group and older email address (which was read only by Johan) are now obsolete.
+
+### New features
+
+While most of the changes to HGVS Nomenclature were operational, we did introduce a few new features.
+
+- **Introduced formal grammar for HGVS Nomenclature syntax** HGVS Nomenclature previously used human-readable descriptions of the syntax of variants. These descriptions were not usable by computers and sometimes conflicted. We have started to introduce the use of formal grammars to represent variant syntax. See [Grammar Elements](../../recommendations/grammar/) for details.
+- **Summarized all syntax in a single table** As a by-product of migrating to a formal grammar, we have also generated a summary of all HGVS syntax on a single page by variant type. See [HGVS Syntax Summary](/recommendations/summary/).
+- **Added a list of software that manipulates HGVS expressions** See [Software](/software).
```

Comment:
The gene fusion work was a big new feature, adding here. There are probably other syntax-related features we will want to include in this list.

```suggestion
- **Added a list of software that manipulates HGVS expressions** See [Software](/software).
- **Updated recommendations for gene fusion syntax** We have aligned definitions of terms used in the characterization of gene fusions and DNA rearrangements with the VICC Gene Fusion Specification, the HUGO Gene Nomenclature Committee, and the ISCN Nomenclature Committee. This has resulted in the revision of the term _fusion transcript_ to _adjoined transcript_ in relevant sections of the nomenclature and glossary. We have also updated our recommendations for users to consult the [VICC Gene Fusion Specification nomenclature](https://fusions.cancervariants.org/en/latest/nomenclature.html) for exon-based representations of adjoined transcripts.
```